src/lib/components/linechart/LineChart.component.js

```diff
@@ -12,6 +12,9 @@ type Props = {
   width?: number,
   height?: number,
   displayTrendLine?: boolean,
+  strokeDashConfig: Object,
```

Comment:
```suggestion
  strokeDashEncodingConfig?: Object,
```
Those new props should be optional IMO.
And I would rename it with `strokeDashEncodingConfig` to indicate this is the strokeDash channel for Encoding Property.